### ISSUE TITLE
Failed to execute process '/usr/local/bin/ga-cmd'

### ISSUE THREAD

**Issue by minhnh94** [arcanericky/ga-cmd#39]
`Failed to execute process '/usr/local/bin/ga-cmd'. Reason:
exec: Exec format error
The file '/usr/local/bin/ga-cmd' is marked as an executable but could not be run by the operating system.`

I did every step mentioned in the Quick start guide but it couldn't go beyond that error.
I was using MacOs

**Comment by arcanericky** [comment]
tldr; see the last paragraph for a better solution.

### Releases for Linux

The *releases* for this project are only for Linux. As stated in the first sentence in the README, "It's like Google Authenticator, but on a command line **using your Linux box.**"

### Binary for Darwin

I've accepted at least a few pull requests for building this on macOS. You can build this on your system and use the resulting binaries. If there are problems doing this, you're welcome to correct them and submit a PR for review.

I'm guessing you don't want to bother with this or are not the developer type. No problem. See below.

### Alternatives

Under the [*Better Alternative*](https://github.com/arcanericky/ga-cmd#the-better-alternative) section in the README, I point to [my TOTP project](https://github.com/arcanericky/totp) and list the reasons for using it instead. It is written in Go and the releases contain a binary for Darwin/macOS which will solve your problem. There are also releases for other operating systems and it's more user friendly too. I'm more active in maintaining it and more likely to fix bugs there.

[Go here for the my TOTP project](https://github.com/arcanericky/totp)
